aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/FilterPanel.js

```diff
@@ -0,0 +1,52 @@
+import { useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBoxGroup,
+  Collapsible,
+  ResponsiveContext,
+} from 'grommet';
+import { FormDown, FormNext } from 'grommet-icons';
+
+import { CATEGORIES } from './constants';
+
+export const FilterPanel = ({ setFilters }) => {
+  const breakpoint = useContext(ResponsiveContext);
+  const [expand, setExpand] = useState(false);
+
+  const filterOptions = (
+    <Box
+      align="start"
+      background="background-front"
+      gap="small"
+      pad={{ right: 'small', vertical: 'small' }}
+      round="xsmall"
+    >
+      <>
+        <Button
+          label="Categories"
+          icon={expand ? <FormDown /> : <FormNext />}
+          onClick={() => setExpand(!expand)}
+        />
+        <Collapsible open={expand}>
+          <CheckBoxGroup
+            options={CATEGORIES}
+            onChange={({ value }) => {
+              setFilters({ categories: value });
+            }}
+          />
+        </Collapsible>
+      </>
+      <Button label="Publishers" icon={<FormNext />} onClick={() => {}} />
+      <Button label="Delivery Methods" icon={<FormNext />} onClick={() => {}} />
+      <Button label="Pricing Models" icon={<FormNext />} onClick={() => {}} />
+    </Box>
+  );
+
+  return ['xsmall', 'small'].includes(breakpoint) ? null : filterOptions;
```

Comment:
is this the pattern to not show the filter in smaller screens when I was putting the screen smaller I didn't expect it to disappear

Reply:
for the time being.

Instead of null, it should serve a filter toolbar button which opens a layer containing the filters. I was running into a couple issues targeting the layer on the desired ref. Need to revisit.

Reply:
got it thank you for clarifying I wasn't sure

Reply:
done